plugins/source/aws/client/multiplexers.go

```diff
@@ -12,15 +10,17 @@ var AllNamespaces = []string{ // this is only used in applicationautoscaling
 }
 
 // Extract region from service list
-func getRegion(regionalMap map[string]*Services) string {
-	if len(regionalMap) == 0 {
+func getRegion(regionalList []string) string {
+	// We should try and find the closest region if possible. This will require checking the following locations:
```

Comment:
I don't quite understand why the comment is so long but the function is so short; what am I missing?

Reply:
Sorry, that should have a `TODO` before of it. Didn't want to add all of that new logic in this PR